00XX-hst-realignment.md

```diff
@@ -0,0 +1,116 @@
+
+Based on historical voting records, the total supply of HNT involved in the [vote is around 10%](https://discord.com/channels/404106811252408320/1093953561970352309/1096572615088427088), which means that HST holders have governance control over the network whenever they choose due to their control of the circulating supply.
+HST affiliated wallets have so far declined to participate in votes as a matter of principle, and so if they choose to participate in this specific vote, that will be strong evidence to indicate that governance has already been captured.
+
+## 3. Incentives for Infrastructure Providers (Miners)
+
+The real value in a DePIN (Decentralized Physical Infrastructure Network) comes from economic incentives to deploy the infrastructure.
+At the time of writing, both the IOT and MOBILE subDAO deployed infrastructure is only about 40% online, declining over time and correlated with a drop in the fiat denominated value of HNT. While the network should be prudent and avoid reacting to short-term, fiat-denominated price fluctuations, ignoring the incentive mechanisms that drive physical infrastructure deployment is dangerous. There
+must be good faith that the network prioritizes the infrastructure providers, as they are the foundation of the Helium
+value chain.
+
+By enacting the proposed distribution change, the amount of HNT flowing to the subDAO treasuries to reward miners would increase by
+about 36,205HNT per day, or 13,215,000HNT per year starting with the halving [beginning August 2023](https://github.com/helium/HIP/blob/main/HIP-solana-parameters/token-emissions-as-of-solana-migration.pdf).
+
+# Implementation
+
+## Prior HIPs
+
+[HIP-20](https://github.com/helium/HIP/blob/main/0020-hnt-max-supply.md) addressed many of the
+concerns and existential risk to the network in not properly incentivizing miners to provide wireless 
+coverage. Without sufficient emissions allocated to miners, when the price of HNT drops, the economic
+incentives to continue mining become strained. This damages the core value proposition of the network.
+However, Net Emissions specifically does not address the overall distribution among stakeholders. 
+
+[HIP-51](https://github.com/helium/HIP/blob/main/0051-helium-dao.md) was the foundational HIP that has led
+to the creation of the Network of Networks model that is the current evolution of the Helium network. It too
+fails to address the distribution HNT emissions and maintains that HST holders are on equal footing as entire
+subDAOs. 
+
+
+## Technical
+
+The technical implementation of this HIP would involve modifying the [`hst_emission_schedule: Option<Vec<PercentItem>>`](https://github.com/helium/helium-program-library/blob/6e2071035a16d419a5f581cb016a94a3cc479f04/programs/helium-sub-daos/src/instructions/update_dao_v0.rs#L9)
+in the `update_dao_v0.rs` instruction within the `helium-program-library`. The value would have to be updated to
+reflect the new 5% emission for HST holders.
```

Comment:
Need to add a few additional sections:

- Drawbacks
- Rationale and Alternatives (this would be a great place to elaborate on the Prior HIPs section you added. Why are those other HIPs important and what rationale storyline does that create?
- Unresolved Questions
- Deployment Impact
- Success Metrics

`# Implementation` can stay as is or go under the higher category ` # Deployment Impact `.